site/app/Resources/views/notification/notification_list.html.twig

```diff
@@ -0,0 +1,32 @@
+{# Render notifications #}
+{% if mgilet_notification_unseen_count(app.user) != 0 %}
+  {% for notification in notificationList|filter(notification => notification.seen == false) -%}
+    {#Render a unseen notification #}
+    {% if (notification.seen == false) %}
+      <ul class="unseen-notifications" id="notification-{{ notification.notification.id }}">
```

Comment:
Please don't misuse ul/li as generic block element here and in the base.html.twig. Use lists when there are actually things to list (like a list of notifications), but use <div> if you only have blocks next to each other.

Reply:
@imphil I had some checks on how others do it and this seems to be a general case(for bootstrap > 3.0). We just have a twig renderer that contains <li> items. <li> can have a nested <ul> and then <li> inside as per documentation. Please have a small look at this [Bootstrap snippet](https://bootsnipp.com/snippets/1e8Ar). So, are we misusing it still?

Reply:
ul/li is semantic markup for a list. A list of notifications is a list. A list of elements in a navigation bar is a list. Use ul/li there. But the code you have here is not a list: it's just different paragraphs: a subject, a message, a date, a divider. That's not a list, and therefore it should not use ul/li.

Reply:
@imphil Sounds logical indeed, I'm using `<span>` and `<div>`s